tests/cross_fw/install/install_checks_torch.py

```diff
@@ -54,3 +55,17 @@
 else:
     msg = f"Invalid execution type {execution_type} (expected 'cpu' or 'gpu')!"
     raise nncf.ValidationError(msg)
+
+EXCLUDED_MODULES_PATTERNS = (
+    "nncf\\.openvino.*",
+    "nncf\\.tensorflow.*",
+    "nncf\\.onnx.*",
+    "nncf\\.experimental\\.tensorflow.*",
+    "nncf\\.experimental\\.openvino.*",
+    "nncf\\.experimental\\.onnx.*",
+    ".*?openvino_[^\\.]*",
```

Comment:
Logically, we should not exclude files from the target backend, even if they contain names of other backends. I would suggest to modify such patterns the following way:
```
"^(?!nncf(?:\\.experimental)*\\.torch.*?\\.).*?openvino_[^\.]*\\.py"
```